gatsby-node.js

```diff
@@ -124,23 +118,29 @@ exports.onPostBuild = async function (
         `query ${i}: found ${nbMatchedRecords} existing records`
       );
 
+      if(objects.length > 0 && !objects[0].objectID && objects[0].id){
```

Comment:
in theory we should do this check for every case, so I think it's actually already duplicating with the fix you did later, so this code isn't needed